engine/execution/convert.go

```diff
@@ -5,14 +5,15 @@ import (
 
 	"github.com/onflow/flow-go/model/convert"
 	"github.com/onflow/flow-go/model/flow"
+	storagemodel "github.com/onflow/flow-go/storage/model"
 )
 
 func GenerateExecutionResultAndChunkDataPacks(
 	prevResultId flow.Identifier,
 	startState flow.StateCommitment,
 	result *ComputationResult) (
 	endState flow.StateCommitment,
-	chdps []*flow.ChunkDataPack,
+	chdps []*storagemodel.StoredChunkDataPack,
```

Comment:
Do we want to keep `StoredChunkDataPack` here? 
My initial though was that we can simply keep using `ChunkDataPack` everywhere and only have `StoredChunkDataPack` as internal badger storage object

Reply:
Right, I am going to keep the `StoredChunkDataPack` as an internal badger object. This is just at the draft state to examine the surface of the change and make sure that it is not breaking any logic.